File: assertion/tests/c/question4/normal.c
#include <stdio.h>

// `findHeaterPower` is function that accepts an argument as long and
// returns an integer as its output.
int findHeaterPower(long input)
{
    int result = 0;
    while (input != 0)
    {
        result += input % 10;
        input /= 10;
    }

    return result;
}

File: assertion/tests/php/question4/normal.php
// `findHeaterPower` is function that accepts an argument as long and
// returns an integer as its output.
function findHeaterPower($input) {
    $s = strval($input);
    $sum = 0;
    for ($i = 0; $i < strlen($s); $i++) {
        $sum += intval($s[$i]);
    }

    return $sum;
}